### ISSUE TITLE
scheduling resize

### ISSUE THREAD

**Issue by hasyee** [tomchentw/react-google-maps#166]
How to be sure what's the appropriate time for triggering resize?
Now I use a similar code:
```
componentDidMount() {
   setTimeout(() => {
      const mapContainer = ReactDOM.findDOMNode(this.refs.loader);
      google.maps.event.trigger(mapContainer, "resize");
   }, 1000);
}
```
I've found that below 1000 ms it won't work, but sometimes that's not enough.

**Comment by tomchentw** [comment]
Will fix this in #161

**Comment by tomchentw** [comment]
Released [v4.7.0](https://github.com/tomchentw/react-google-maps/releases/tag/v4.7.0)

**Comment by hasyee** [comment]
Thx!!
